Doc 1246/preview docs demo (#390)
swan-io/docs: docs/preview/belgian-ibans.mdx

---

> title: Belgian IBANs

Reviewer: We haven't had an IBANs page in the past for previous IBAN launches - will the intention of this be that once it's live, we'll build all these updates into their relevant pages?

Author: Yes, that was my thinking :)

timeline: November 2025
status: upcoming
description: Issue Belgian IBANs for account holders and members located in Belgium, expanding Swan's local IBAN coverage across Europe.
---

import PreviewInfo from './partials/_preview-info.mdx';

<PreviewInfo />

# Belgian IBANs

Issue local Belgian IBANs for your account holders and members, providing enhanced banking experience and regulatory compliance for Belgian users.

## Overview

> Belgian IBAN support expands Swan's local IBAN coverage to Belgium, allowing you to offer familiar local banking experiences to Belgian customers. Account holders registered in Belgium will receive Belgian IBANs (with a `BE` prefix) instead of French IBANs.

Reviewer: Why do we call it "Belgian IBAN support"? Can we just say - "Offering Belgian IBANs expands Swan's local IBAN coverage,...

Also don't think we need to mention the country so many times in the sentence.

> ### Key benefits

Reviewer: Some of this is quite marketingish - will probably need a PMM to review to make sure it aligns with their value prop messaging.

Author: Sounds good! Any name in mind? I could send them a preview link

- **Expanded market reach**: Serve Belgian customers with local banking infrastructure.
- **Regulatory alignment**: Meet Belgian banking regulations and reporting requirements.
- **Enhanced user trust**: Local IBANs increase customer confidence and reduce payment friction.
- **Local banking integration**: Seamless integration with Belgian banking systems and practices.

## Geographic scope

Belgian IBANs are available for:
- **Account holders** registered in Belgium.

> - **Account members** located anywhere globally (standard Swan coverage).

Reviewer: Would reword this - as its not clear if its located anywhere - or whether its based on where Swan covers.

- **Business accounts** for Belgian companies and subsidiaries.

## Integration impact

Belgian IBAN support maintains full compatibility with existing Swan integrations:

> - **No breaking changes**: Existing API calls continue to work unchanged.

Reviewer: Would go with either "continue to work" or "remain unchanged" 

i.e. 
Existing API calls continue to work. 

Or 

Existing API calls remain unchanged. 

Saying both in one sentence doesn't add anything.

- **Standard onboarding flow**: Uses existing individual and company onboarding processes.
- **Consistent permissions**: Account membership permissions work identically across all local IBANs.

### Onboarding requirements

Belgian account holders follow Swan's standard onboarding requirements with these specifics:

<FieldRequirementsLegend />

**Individual accounts:**

| API field | Requirement | Notes |
|---|:---:|---|
| `accountCountry` | <Req /> | Must be Belgium |
| `email` | <Req /> | Standard verification |
| `employmentStatus` | <Req /> | Required for compliance |
| `monthlyIncome` | <Req /> | Belgian KYC requirement |

> | `residencyAddress` | <Req /> | Full Belgian address required |

Reviewer: Should specify that its a Full Belgian residency address"

| `taxIdentificationNumber` | <Opt /> | Optional |

**Company accounts:**

| API field | Requirement | Notes |
|---|:---:|---|
| `accountCountry` | <Req /> | Must be Belgium |

> | `businessActivity` | <Req /> | Belgian business registration |

Reviewer: Is there a specific name for this? e.g. the place that they register the businesses?

| `companyType` | <Req /> | Standard company verification |

> | `registrationNumber` | <Cnd /> | If registered in Belgium |

Reviewer: Don't they have to be registered in Belgium?

Author: I don't think they have to, we have a field for that (`isRegistered`)

> | `residencyAddress` | <Req /> | Belgian business address |

Reviewer: Residency address is the field - even for business addresses?

Author: Likewise, this isn't clear in the documents I have. All I can do is make it another vague placeholder and ask a SME

---
title: Belgian IBANs
timeline: November 2025
chronology: 15-11-2025
status: upcoming
description: Issue Belgian IBANs for account holders and members located in Belgium, expanding Swan's local IBAN coverage across Europe.
---

import PreviewInfo from './partials/_preview-info.mdx';

<PreviewInfo />

# Belgian IBANs

Issue local Belgian IBANs for your account holders and members, providing enhanced banking experience and regulatory compliance for Belgian users.

## Overview

> Belgian IBAN support expands Swan's local IBAN coverage to Belgium, allowing you to offer familiar local banking experiences to Belgian customers. Account holders registered in Belgium will receive Belgian IBANs (with a `BE` prefix) instead of French IBANs.

Author: ```suggestion
Offering Belgian IBANs expands Swan's local IBAN coverage, allowing you to offer familiar local banking experiences to end-users. Account holders registered in Belgium will receive Belgian IBANs (with a `BE` prefix) instead of French IBANs.
```

### Key benefits

- **Expanded market reach**: Serve Belgian customers with local banking infrastructure.
- **Regulatory alignment**: Meet Belgian banking regulations and reporting requirements.
- **Enhanced user trust**: Local IBANs increase customer confidence and reduce payment friction.
- **Local banking integration**: Seamless integration with Belgian banking systems and practices.

## Geographic scope

> Belgian IBANs are available for:
> - **Account holders** registered in Belgium.
> - **Account members** located anywhere globally (standard Swan coverage).

Author: ```suggestion
Belgian IBANs are available for:
- **Account holders** registered in Belgium.
- **Account members** in Swan-supported European markets.
```

- **Business accounts** for Belgian companies and subsidiaries.

## Integration impact

Belgian IBAN support maintains full compatibility with existing Swan integrations:

> - **No breaking changes**: Existing API calls continue to work unchanged.

Author: ```suggestion
- **No breaking changes**: Existing API calls continue to work.
```

- **Standard onboarding flow**: Uses existing individual and company onboarding processes.
- **Consistent permissions**: Account membership permissions work identically across all local IBANs.

### Onboarding requirements

Belgian account holders follow Swan's standard onboarding requirements with these specifics:

<FieldRequirementsLegend />

**Individual accounts:**

| API field | Requirement | Notes |
|---|:---:|---|
| `accountCountry` | <Req /> | Must be Belgium |
| `email` | <Req /> | Standard verification |
| `employmentStatus` | <Req /> | Required for compliance |
| `monthlyIncome` | <Req /> | Belgian KYC requirement |

> | `residencyAddress` | <Req /> | Full Belgian address required |

Author: ```suggestion
| `residencyAddress` | <Req /> | Full Belgian residency address required |
```

| `taxIdentificationNumber` | <Opt /> | Optional |

**Company accounts:**

| API field | Requirement | Notes |
|---|:---:|---|
| `accountCountry` | <Req /> | Must be Belgium |

> | `businessActivity` | <Req /> | Belgian business registration |

Author: ```suggestion
| `businessActivity` | <Req /> | Business activity classification |
```

Author: I had to work from notion documents and had no clear task list so it's all very frankenstein-y. 

afaict, looking through all the documents again, I have no further information on the specific `businessActivity` contents. Do you know who's the PM or another right person to ask? @racene-swan

Reviewer: Martin Vallade is the PM - definitely hit him up with the questions :)

| `companyType` | <Req /> | Standard company verification |

> | `registrationNumber` | <Cnd /> | If registered in Belgium |

Author: ```suggestion
| `registrationNumber` | <Cnd /> | Required if the company [is registered](/topics/onboarding/company/#country-reqs-company-account-holder) |
```

> | `residencyAddress` | <Req /> | Belgian business address |

Author: ```suggestion
| `residencyAddress` | <Req /> | Business address |
```
